src/Error.php

```diff
@@ -30,8 +32,13 @@ public function getMessage(): string
     /**
      * @psalm-return list<int|string>
      */
-    public function getValuePath(): array
```

Comment:
Why remove value path?

Reply:
Because it into `$attribute`

Reply:
This is don't cover all cases. In master special make value path (see discussion in team chat).